package.json

```diff
@@ -1,6 +1,6 @@
 {
     "devDependencies": {
-        "@rocket.chat/apps-engine": "WideChat/Rocket.Chat.Apps-engine.git#v1.29.2",
+        "@rocket.chat/apps-engine": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#dialogflow_branch",
```

Comment:
NOTE: this is fine for testing, but we need to be careful that we revert of fix this before we merge.